app/models/viewModels/ChargeDetailsViewModel.scala

```diff
@@ -16,10 +16,19 @@
 
 package models.viewModels
 
+import play.api.i18n.Messages
+import views.templates.payments.PaymentMessageHelper
+
 import java.security.MessageDigest
 
 trait ChargeDetailsViewModel {
+  val chargeType: String
   val outstandingAmount: BigDecimal
+  val overdue: Boolean = false
+
+  def title(implicit messages: Messages): String = messages(PaymentMessageHelper.getChargeType(chargeType).title)
```

Comment:
Could you move the tests around for this? It's currently being tested in a few of the subclass view model specs (looks to be all the LPP models?) but I think following this change it makes more sense to remove those repeats and just have one test for it in `ChargeDetailsViewModelSpec`.